index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio || BB Simon</title>
+
+  <!-- Custom CSS -->
+  <link rel="stylesheet" href="./styles/style.css">
+
+  <!-- Google fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap"
+    rel="stylesheet">
+
+</head>
+<body>
+  <!-- Navigation bar -->
+  <header>
+    <nav class="navbar">
+      <ul class="nav-items-wrapper">
+        <li class="nav-item">
+          <a class="logo-wrapper" href="/">
+            <img src="./assets/My Logo.png" alt="Logo">
+          </a>
+        </li>
+        <li class="nav-item">
+          <img src="./assets/Icon - Menu.png" alt="Menu">
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <!-- Main content -->
+  <main>
+    <section>
+      <header class="header">
+        <h1 id="title">I’m Simon
+          <br />Glad to see you!
+        </h1>
+        <p id="description">
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h4 id="title-contact">
+          LET’S CONNECT
+        </h4>
+        <ul class="icons-wrapper">
```

Comment:
[OPTIONAL] Kindly fix the position of the icons. They shouldn't be overflowing through section. And there should be enough place under it.
Your design:
<img width="407" alt="Screenshot 2022-12-02 at 12 19 02 AM" src="https://user-images.githubusercontent.com/25887253/205162214-3bc1f2b8-976f-4543-ae50-88b7c9c3cc5d.png">

Figma template:
<img width="407" alt="Screenshot 2022-12-02 at 12 26 32 AM" src="https://user-images.githubusercontent.com/25887253/205162510-afa81eb4-ee7a-4b38-bfc6-0a1149b5ee29.png">